src/main/scala/org/phenoscape/kb/KBVocab.scala

```diff
@@ -17,6 +17,7 @@ object KBVocab {
   val BDRank = IRI.create(s"${BDSearchPrefix}rank")
   val rdfsLabel = ObjectProperty(RDFS_LABEL.getIRI)
   val rdfsSubClassOf = ObjectProperty(RDFS_SUBCLASS_OF.getIRI)
+  val transitiveProperty = ObjectProperty("http://www.w3.org/2002/07/owl#TransitiveProperty")
```

Comment:
It doesn't really matter, since you're just using this to embed the IRI in SPARQL, but `TransitiveProperty` is not an ObjectProperty. I would just create this as an IRI.